gst/nnstreamer/tensor_query/nnstreamer_edge.h

```diff
@@ -0,0 +1,165 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
```

Comment:
It's good now. but when you've migrated to nnstreamer-edge.git or aitt.git, you may use Apache-2.0

Reply:
marked todo (update license)